src/main/java/frc/robot/Robot.java

```diff
@@ -18,6 +18,8 @@
 public class Robot extends TimedRobot {
   private Command m_autonomousCommand;
 
+  boolean hasAutonomousRun;
```

Comment:
this is never set to true
https://github.com/FRCTeam3255/2024_Offbot_Code/blob/dbdd2afe963247d7a9535f7752eb58f633c5b246/src/main/java/frc/robot/Robot.java#L97-L98